README.md

```diff
@@ -23,18 +23,18 @@ bun install stealth-address-sdk
 
 ## Testing
 
-To run tests using a local fork of a chain using an rpc url
+Tests default to using your local [anvil](https://book.getfoundry.sh/anvil/) node
 
 ```bash
-bun run anvil-local
+anvil
 bun run test
 ```
 
-To run tests using the remote chain using an rpc url
+Alternatively, run your tests using a fork of your provided (`TEST_RPC_URL` in `env`) rpc url
 
 ```bash
 bun run anvil-remote
```

Comment:
this script doesn't exist

Reply:
Also even if it is `anvil-remote`, I'm getting failures at bun run test-fork because `ChainMismatchError: The current chain of the wallet (id: 11155111) does not match the target chain for the transaction (id: 31337 – Foundry).`

Reply:
should be handled by: https://github.com/ScopeLift/stealth-address-sdk/pull/37/commits/040496e7b14693823574185490c2d0776d87d349